charts/clusterapi-resources/templates/ConfigMapAddClusterToArgo.yaml

```
@@ -0,0 +1,117 @@
+{{- if .Values.argo.enabled }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: add-cluster-to-argo
+  namespace: {{ .Release.Namespace }}
+data:
+  data: |-
+    {{- if .Values.argo.serviceAccount.create }}
+    ---
+    apiVersion: v1
+    kind: ServiceAccount
+    metadata:
+      name: {{ .Values.argo.serviceAccount.name }}
+      namespace: {{ .Values.argo.namespace }}
+    ---
+    apiVersion: rbac.authorization.k8s.io/v1
+    kind: ClusterRoleBinding
+    metadata:
+      name: {{ .Values.argo.serviceAccount.name }}-binding
+    roleRef:
+      apiGroup: rbac.authorization.k8s.io
+      kind: ClusterRole
+      name: cluster-admin
+    subjects:
+    - kind: ServiceAccount
+      name: {{ .Values.argo.serviceAccount.name }}
+      namespace: {{ .Values.argo.namespace }}
+    {{- end }}
+    ---
+    apiVersion: v1
+    kind: Pod
+    metadata:
+      name: initial-cluster-setup
+      namespace: {{ .Values.argo.namespace }}
+      labels:
+        app: alpine
```

Maybe add a more meaningful name to the `app` label.
```suggestion
        app: argo-initial-cluster-setup
```